src/pngchunk_int.cpp

```diff
@@ -64,6 +64,8 @@ namespace Exiv2 {
                                    int*           outWidth,
                                    int*           outHeight)
     {
+        assert(data.size_ >= 8);
```

Comment:
I think it is enough for the moment, but if in the future we want to add a unit tests for this method, we will need to replace the assertion by a run-time check. Note that it might be interesting to run the unit tests on Debug mode, and asserts will cause an abort in the condition is met. 

Instead of using asserts we could use run-time checks and throw exceptions. Then we can add test-cases covering such cases.

Reply:
Yes, we should add a better check in the future. Currently this is ok, since `PngChunk::decodeIHDRChunk()` is only called from one place, where I added an explicit check that `data.size_` is larger 8.

The better check will probably be introduced by a redesign of the `DataBuf` class (see #218).